### ISSUE TITLE
Can Quarto files be added to the file type list?

### ISSUE THREAD

**Issue by RaymondBalise** [r-lib/styler#976]
When I try `styler:::style_active_file()` on my Quarto files (that contain R code) I get a message saying:

```
Can only style .R, .Rmd and .Rnw files.
```

It would be wonderful if .qmd files could be styled.  Renaming the files the the Rmd suffix does work but it is a bit of a hassle.

**Comment by lorenzwalthert** [comment]
I agree. We added support for quarto docs for the other APIs, not sure why not for the addin.

**Comment by lorenzwalthert** [comment]
Actually you just need the development version of {styler}.

**Comment by lorenzwalthert** [comment]
Closed with #931

**Comment by RaymondBalise** [comment]
Thank you kindly.  I should have checked the news.md.
